demo/api/schema.gql

```diff
@@ -902,6 +903,11 @@ enum ProductTagSortField {
   updatedAt
 }
 
+input TranslationInput {
```

Comment:
We should prefix this input with AzureAi to avoid name conflicts with other translation services

Reply:
a53e44842567e4c67db107a79418e3ae95139072